src/main/java/seedu/address/logic/commands/FindCommand.java

```diff
@@ -16,19 +21,37 @@
 
     public static final String MESSAGE_USAGE = COMMAND_WORD + ": Finds all properties whose names contain any of "
             + "the specified keywords (case-insensitive) and displays them as a list with index numbers.\n"
```

Comment:
This does not seem very reflective of the new find function, can we make it clearer?